fifeforspark/utils.py

```diff
@@ -0,0 +1,160 @@
+from pyspark.sql import SparkSession
+import pandas as pd
+import findspark
+import pyspark.sql
+from pyspark.mllib.random import RandomRDDs
+import numpy as np
+from pyspark.sql.types import StructType, StructField, StringType, IntegerType, FloatType
+import random as rn
+import argparse
+
+findspark.init()
+sc = SparkSession.builder.getOrCreate()
+
+def create_example_data1(n_persons: int = 3, n_periods: int = 12
+) -> pd.core.frame.DataFrame:
+    schema = StructType([
+        StructField('individual', StringType(), True),
+        StructField('period', StringType(), True),
+        StructField('feature_1', StringType(), True),
+        StructField('feature_2', StringType(), True),
+        StructField('feature_3', StringType(), True),
+        StructField('feature_4', StringType(), True)])
+    values = sc.createDataFrame([], schema)
+    for i in np.arange(n_persons):
+            period = np.random.randint(n_periods)+1 #Sparkify this
+            rdd1 = RandomRDDs.uniformRDD(sc,size = 1, seed = 9999)
+            x_1 = rdd1.first()
+
+            obj1 = sc.sparkContext.parallelize(["A","B","C"])
+            x_2 = obj1.takeSample(False, 1, seed = 9999)[0]
+
+            rdd3 = RandomRDDs.uniformRDD(sc, size = 1, seed = 9999).map(lambda v: 1 + v)
```

Comment:
This may be a poor question on my part- but why are we using RDDs more than the actual Spark DataFrames? I see that you return it as a DataFrame but why not start it out that way too?

Reply:
You're right for obj1, I could initiate this as a DF instead then take a sample from that. For the RandomRDDs functions, they return RDD objects by default, so I'm not sure how to convert them to DFs.